app/utils/transaction-view.js

```diff
@@ -176,12 +176,9 @@ function getCurrentPageNumber (connectorData) {
 
 function getCurrentPageSize (connectorData) {
   const selfLink = connectorData._links && connectorData._links.self
-  let queryString
-  let limit
-
   if (selfLink) {
-    queryString = url.parse(selfLink.href).query
-    limit = Number(qs.parse(queryString).display_size)
+    const searchString = new URL(selfLink.href).search.substring(1)
```

Comment:
Worth check for null before substring(1) ?

Reply:
If there aren't any search params the URL object contains an empty string for the search key, `.substring(1)` on an empty string will return an empty string.